src/App.css

```diff
@@ -0,0 +1,292 @@
+/* Importing My CSS Component Library Ingredients  */
+@import url("https://ingredient.netlify.app/css/main.css");
+/* Getting Google Fonts  */
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;600;700;800&display=swap');
+
```

Comment:
you can link them in the index.html file instead, so that we use features like preconnect to improve performance. importing in CSS is slow

Reply:
Got it. Thanks, that makes sense. Will do it.